examples/bridge/sn/src/dojo_bridge.cairo

```diff
@@ -0,0 +1,192 @@
+use starknet::{ContractAddress, ClassHash};
+use dojo::world::IWorldDispatcher;
+
+#[starknet::interface]
+trait IDojoBridge<TState> {
+    // IWorldProvider
+    fn world(self: @TState, ) -> IWorldDispatcher;
+
+    // IUpgradeable
+    fn upgrade(ref self: TState, new_class_hash: ClassHash);
+
+    // WITHOUT INTERFACE !!!
+    fn initializer(ref self: TState, l1_bridge: felt252, l2_token: ContractAddress);
+    fn initiate_withdrawal(ref self: TState, l1_recipient: felt252, amount: u256);
+    fn get_l1_bridge(self: @TState, ) -> felt252;
+    fn get_token(self: @TState, ) -> ContractAddress;
+    fn dojo_resource(self: @TState, ) -> felt252;
+}
+
+///
+/// Model
+///
+
+#[derive(Model, Copy, Drop, Serde)]
+struct DojoBridgeModel {
+    #[key]
+    token: ContractAddress,
+    // address L1 bridge contract address, the L1 counterpart to this contract
+    l1_bridge: felt252,
+    // Dojo ERC20 token on Starknet
+    l2_token: ContractAddress
```

Comment:
Are `token` and `l2_token` the same and `token` is only a key or what's the meaning of `token`?

Reply:
`token` is misleading indeed, should be `bridge` is the bridge contract address